library/flrtvc.py

```diff
@@ -165,9 +165,9 @@ def remove_efix(output):
     res = 0
     logging.debug(': Removing all installed efix')
 
-    cmd = ['"export LC_ALL=C; rc=0;'
-            ' for i in `/usr/sbin/emgr -P |/usr/bin/tail -n +4 |/usr/bin/awk \'{print \$NF}\'`;'
-            ' do /usr/sbin/emgr -r -L $i || (( rc = rc | $? )); done; echo rc=$rc"']
+    cmd = [r'"export LC_ALL=C; rc=0;'
```

Comment:
I think the inclusion of the 'r'  is a typo, right?

Reply:
No this was to avoid the following complaint from Flake8:
 W605 invalid escape sequence '\$'
But it does not work, the r should be on the line just after:
    cmd = ['"export LC_ALL=C; rc=0;'
            r' for i in `/usr/sbin/emgr -P |/usr/bin/tail -n +4 |/usr/bin/awk \'{print \$NF}\'`;'
Will fix this along with other complaints in a next PR.